src/api/supabase/queries/cases.ts

```diff
@@ -2,13 +2,20 @@ import { UUID } from 'crypto';
 import supabase from '../createClient';
 
 export async function getAllCases() {
```

Comment:
we might need to update these after the switch to using the database function for the queries. May or may not also need to update the database func?